pymongo/collection.py

```diff
@@ -686,7 +686,9 @@ def insert_many(self, documents, ordered=True,
 
         .. versionadded:: 3.0
         """
-        if not isinstance(documents, abc.Iterable) or not documents:
+        if not isinstance(documents, abc.Iterable) \
```

Comment:
While this isn't incorrect, it is bad style to use backslash. As per the [style guide](https://www.python.org/dev/peps/pep-0008/#maximum-line-length):

> The preferred way of wrapping long lines is by using Python's implied line continuation inside parentheses, brackets and braces. Long lines can be broken over multiple lines by wrapping expressions in parentheses. These should be used in preference to using a backslash for line continuation.

Can you modify this so the conditional check is enclosed in `()` instead?